# Publish to C++ controller to request the current value of a single data_type
# (ex. end effectors, joint angles, etc.)
time stamp
int32 id  # ID must be echoed back in SampleResult
int32 arm
